minard/templates/check_rates_history.html

```diff
@@ -0,0 +1,162 @@
+                <option value="10">10</option>
+                <option value="11">11</option>
+                <option value="12">12</option>
+                <option value="13">13</option>
+                <option value="14">14</option>
+                <option value="15">15</option>
+            </select>
+            /
+            <select id="channel-sel">
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+                <option value="6">6</option>
+                <option value="7">7</option>
+                <option value="8">8</option>
+                <option value="9">9</option>
+                <option value="10">10</option>
+                <option value="11">11</option>
+                <option value="12">12</option>
+                <option value="13">13</option>
+                <option value="14">14</option>
+                <option value="15">15</option>
+                <option value="16">16</option>
+                <option value="17">17</option>
+                <option value="18">18</option>
+                <option value="19">19</option>
+                <option value="20">20</option>
+                <option value="21">21</option>
+                <option value="22">22</option>
+                <option value="23">23</option>
+                <option value="24">24</option>
+                <option value="25">25</option>
+                <option value="26">26</option>
+                <option value="27">27</option>
+                <option value="28">28</option>
+                <option value="29">29</option>
+                <option value="30">30</option>
+                <option value="31">31</option>
+            </select>
+            <button type=button onclick="history();">Update Plot</button>
+        </div>
+    </div>
+    <h1> </h1>
+    <div class="row">
+        <div class="col-md-4">
+            {% if stats and discriminator_threshold %}
+            <table class="table table-bordered">
+                <tr>
+                    <th>Mean</th> 
+                    <th>Minimum</th> 
+                    <th>Maximum</th>
+                    <th>VThr</th> 
+                </tr>
+                <tr>
+                    <td> {{ stats[2] }} &plusmn; {{ stats[3] }} </td>
+                    <td> {{ stats[1] }} </td>
+                    <td> {{ stats[0] }} </td>
+                    <td> {{ discriminator_threshold['vthr'][channel] - discriminator_threshold['zero_disc'][channel] }} </td>
+                </tr>
+            </table>
+            {% endif %}
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12" id="main">
+            {% if data %}
+                <script type="text/javascript" src="js/scatter_plot_rates.js"></script>
```

Comment:
Is this meant to be here? You include it at the bottom. If it is meant to be here, it should be:

```javascript
<script src="{{ url_for('static', filename='js/scatter_lot_rates.js') }}"></script>
```